scripts/ci/pylint_todo.txt

```diff
@@ -66,12 +66,10 @@
 ./airflow/contrib/operators/jenkins_job_trigger_operator.py
 ./airflow/contrib/operators/jira_operator.py
 ./airflow/contrib/operators/kubernetes_pod_operator.py
-./airflow/gcp/operators/mlengine.py
 ./airflow/contrib/operators/mongo_to_s3.py
 ./airflow/contrib/operators/opsgenie_alert_operator.py
 ./airflow/contrib/operators/oracle_to_azure_data_lake_transfer.py
 ./airflow/contrib/operators/oracle_to_oracle_transfer.py
-./airflow/contrib/operators/pubsub_operator.py
```

Comment:
Is this file already pytlint compatible?

Reply:
Yes, PubSub has already been moved to core.